libsolidity/analysis/NameAndTypeResolver.h

```diff
@@ -90,6 +90,9 @@ class NameAndTypeResolver: private boost::noncopyable
 		std::vector<Declaration const*> const& _declarations
 	);
 
+	/// Generate and store warnings about variables that are named like instructions.
+	void warnVariablesNamedLikeInstructions(void);
```

Comment:
In C, in declarations, `()` would mean there might be arguments, might not be arguments.

In C++, maybe `()` and `(void)` mean the same thing.  I'll look up Stroustrup's.